source/io/module_javaScriptInvoke.js

```diff
@@ -338,6 +324,19 @@
             };
         };
 
+        var createJavaScriptParametersArray = function (isInternalFunction, parametersPointer, parametersCount) {
+            var parameters = new Array(parametersCount);
```

Comment:
just do `var parameters = [];`

Using the Array constructor does not actually allocate any space. It creates a sparse array and is ultimately more difficult for JS engines to optimize than growing a normal array.

Reply:
Good to know. Done.